app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
+	end
+
+	def print_fib(range)
+		range.each do |n|
+			p fib(n)
+		end
+	end
+
+	private
+		def fib(n)
+			return 1 if n <= 1
+			fib(n-2) + fib(n-1)
```

Comment:
__[development][rubocop]__
* Style/Tab: Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Style/SpaceAroundOperators: Surrounding space missing for operator `-`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)